### DISCUSSION TITLE
Time-based scoring

### DISCUSSION THREAD

**Discussion by mapeje** [mawoka-myblock/ClassQuiz#308] [Ideas]
First, a big thanks for a great project! I would like to request the feature to get more points by answering faster, much like Kahoot or Menti. This will increase the engagement and also easier to announce a winner if many participants end on the same score.

**Comment by mawoka-myblock** [comment]
That's already implemented

**Reply by mapeje** [reply]
Wow, somehow I missed that . Thanks!!
